src/NServiceBus.Core/Serializers/MessageDeserializerResolver.cs

```diff
@@ -27,6 +36,6 @@ public IMessageSerializer Resolve(Dictionary<string, string> headers)
         }
 
         IMessageSerializer defaultSerializer;
-        Dictionary<string, IMessageSerializer> serializersMap;
+        Dictionary<string, IMessageSerializer> serializersMap = new Dictionary<string, IMessageSerializer>();
```

Comment:
We have no real guidance around it. I would have naturally created the dictionary and assigned it to the field in the constructor for better visibility. Not saying doing it like that is wrong, just mentioning to see if there is a learning we can take away